src/detection/detection-card/action/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Button } from '@material-ui/core'
+
+import { CardAction } from 'detection/detection-card/action/container'
+
+export const CardButtons = () => <CardAction>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125415487-7ec6b265-5015-47f4-a2ed-eb7779c420b3.png)

Let's put a little bit of space between these buttons and the bottom of the card.